src/signals/incident/components/IncidentPreview/index.js

```diff
@@ -168,6 +154,7 @@ const IncidentPreview = ({ incident, preview }) => (
 IncidentPreview.propTypes = {
   incident: incidentType.isRequired,
   preview: PropTypes.object,
+  sectionLabels: PropTypes.object,
```

Comment:
`PropTypes.shape({`

Reply:
Updated to use shape for the first level